src/pilot_simulation.jl

```diff
@@ -123,23 +130,214 @@ forcings = ModelForcing(u=Fu′, v=Fv′, w=Fw′, T=Fθ′, S=Fs′)
 const ρ₀ = 1027.0  # Density of seawater [kg/m³]
 const cₚ = 4000.0  # Specific heat capacity of seawater at constant pressure [J/(kg·K)]
 
-@inline wind_stress_x(i, j, grid, t, ũ′, c′, p) = p.ℑτx(t) / ρ₀
-@inline wind_stress_y(i, j, grid, t, ũ′, c′, p) = p.ℑτy(t) / ρ₀
-@inline     heat_flux(i, j, grid, t, ũ′, c′, p) = p.ℑQθ(t) / ρ₀ / cₚ
-@inline     salt_flux(i, j, grid, t, ũ′, c′, p) = p.ℑQs(t)  # FIXME?
+@inline wind_stress_x(i, j, grid, t, I, ũ′, c′, p) = p.ℑτx(t) / ρ₀
+@inline wind_stress_y(i, j, grid, t, I, ũ′, c′, p) = p.ℑτy(t) / ρ₀
+@inline     heat_flux(i, j, grid, t, I, ũ′, c′, p) = - p.ℑQθ(t) / ρ₀ / cₚ
+@inline     salt_flux(i, j, grid, t, I, ũ′, c′, p) =   p.ℑQs(t) / ρ₀  # Minus sign because a freshwater flux would decrease salinity.
 
 u′_bcs = UVelocityBoundaryConditions(grid, top=FluxBoundaryCondition(wind_stress_x))
 v′_bcs = UVelocityBoundaryConditions(grid, top=FluxBoundaryCondition(wind_stress_y))
 θ′_bcs =    TracerBoundaryConditions(grid, top=FluxBoundaryCondition(heat_flux))
-s′_bcs =    TracerBoundaryConditions(grid, top=FluxBoundaryConditions(salt_flux))
+s′_bcs =    TracerBoundaryConditions(grid, top=FluxBoundaryCondition(salt_flux))
+
+#####
+##### Model setup
+#####
 
 model = IncompressibleModel(
     architecture = arch,
     float_type = FT,
     grid = grid,
-    tracrs = (:T, :S),
+    tracers = (:T, :S),
     coriolis = FPlane(latitude=lat),
     boundary_conditions = (u=u′_bcs, v=v′_bcs, T=θ′_bcs, S=s′_bcs),
+    closure = AnisotropicMinimumDissipation(),
     forcing = forcings,
     parameters = (ℑτx=ℑτx, ℑτy=ℑτy, ℑQθ=ℑQθ, ℑQs=ℑQs, ℑU=ℑU, ℑV=ℑV, ℑΘ=ℑΘ, ℑS=ℑS, μ=μ)
 )
+
+#####
+##### Initial conditions
+#####
+
+ε(μ) = μ * randn() # noise
+
+U₀(x, y, z) = ℑU(0, z)
+V₀(x, y, z) = ℑV(0, z)
+W₀(x, y, z) = ε(1e-10)
+Θ₀(x, y, z) = ℑΘ(0, z)
```

Comment:
Is "0" for "time"? Should we use the function signature `z, t`?

Reply:
Yeah `z, t` might be clearer, just haven't flipped it around as xarray returns data with `t, z` (might depend on how NetCDF orders the coordinates but I'm sure it's easy to change).